iele-gas.md

```diff
@@ -601,10 +615,10 @@ The cost to call another contract is very similar to the cost in EVM:
 -   The gas stipend paid to the callee to execute its code.
 
 ```k
-    rule <k> #compute [ _, RETS::LValues = call _ at ACCTTO ( ARGS ) send VALUE , gaslimit GCAP, SCHED ] => Ccall(SCHED, #accountEmpty(ACCTTO), GCAP, GAVAIL, VALUE, #sizeLVals(RETS), intSizes(ARGS)) ... </k>
+    rule <k> #compute [ _, RETS::LValues = call _ at ACCTTO ( ARGS ) send VALUE , gaslimit GCAP, SCHED ] => Ccall(SCHED, #accountEmpty(ACCTTO), GCAP *Int Sgasdivisor < SCHED >, GAVAIL, VALUE, #sizeLVals(RETS), Ccallarg(SCHED, ARGS)) ... </k>
```

Comment:
Shouldn't we divide by `Sgasdivisior` and round at the contract return? If we don't, then the exact fine-grained cost left over could affect whether the caller runs out of gas even when the different amounts would round to the same cost.

Reply:
Can you give me an example of your concern? I'm having trouble following what you see as the problem here.

Reply:
If we are not trying to hide the fine-grained gas costs at inter-contract boundaries then this is fine.